src/store/actionCreators.ts

```diff
@@ -9,7 +9,7 @@ export function addDataBlock(avastarParsedDataBloc: AvastarParsedDataPoint[]) {
   return action;
 }
 
-export function removeArticle() {
+export function removeDataBlock() {
```

Comment:
never used for now? @tangigouez @Laetitia-Saulnier @JeFaisLeCafe

Reply:
Non, pour le moment ce n'est pas utilisé. Mais on va s'en servir dans les prochaines semaines. Je serais favorable à l'implémentation d'un bouton qui permet à l'utilisateur de clear le State pour lui donner plus de contrôle.